airbyte-integrations/connectors/source-zenloop/source_zenloop/components.py

```diff
@@ -8,11 +8,14 @@
 
 from airbyte_cdk.models import SyncMode
 from airbyte_cdk.sources.declarative.stream_slicers.substream_slicer import SubstreamSlicer
-from airbyte_cdk.sources.declarative.types import StreamSlice, StreamState
+from airbyte_cdk.sources.declarative.types import Config, StreamSlice, StreamState
 
 
 @dataclass
 class ZenloopSubstreamSlicer(SubstreamSlicer):
+
+    config: Config
```

Comment:
This was an interesting case. We used to extract the config via options because it was stored there. This seemed like a misuse of the options vs. config separation, instead like other components, `config` can be passed via parameter separately and we retrieve it that way for this custom component